client/app/components/form/MultiStepFormBase.tsx

```diff
@@ -1,11 +1,17 @@
+"use client";
+
+import { useState } from "react";
 import { useParams } from "next/navigation";
+import { Button } from "@mui/material";
 import { RJSFSchema } from "@rjsf/utils";
 import { Alert } from "@mui/material";
 import FormBase from "./FormBase";
 import MultiStepHeader from "./MultiStepHeader";
 import MultiStepButtons from "./MultiStepButtons";
 
 interface MultiStepFormProps {
+  allowBackNavigation?: boolean;
```

Comment:
This wasn't added by me and isn't part of this PR. The only reason this has changed in this PR is because I tend to organize my types/destructure my props in alphabetical order.